<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayerDialog</class>
 <widget class="QDialog" name="MediaPlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>详细信息</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonMediaPlayerPause">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>208</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaPlayerPlay">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>208</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaPlayerReturn">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>400</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaPlayerCall">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>336</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>回呼</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaPlayerStop">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>272</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>停止 </string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>18</y>
     <width>550</width>
     <height>440</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameVideoShow">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>77</y>
     <width>518</width>
     <height>377</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>20</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>,white</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26px, white;
border: 0px;</string>
   </property>
   <property name="text">
    <string>对讲详情 - 1111-2222</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>pushButtonMediaPlayerPause</zorder>
  <zorder>pushButtonMediaPlayerPlay</zorder>
  <zorder>pushButtonMediaPlayerReturn</zorder>
  <zorder>pushButtonMediaPlayerCall</zorder>
  <zorder>pushButtonMediaPlayerStop</zorder>
  <zorder>frameVideoShow</zorder>
  <zorder>labelInfo</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonMediaPlayerPause</tabstop>
  <tabstop>pushButtonMediaPlayerPlay</tabstop>
  <tabstop>pushButtonMediaPlayerStop</tabstop>
  <tabstop>pushButtonMediaPlayerCall</tabstop>
  <tabstop>pushButtonMediaPlayerReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
